### ISSUE TITLE
Mohist does not perform TwitchSpawn events

### ISSUE THREAD

**Issue by NovaPixell** [Teneted/Tenet#545]
**Describe the bug**
I have encountered problems with the mod "TwitchSpawn", which can run events on Minecraft depending on what happens in the Twitch live.

**To Reproduce**
Install the mod on your Mohist Server.
Customize the config with your Minecraft nickname. Mod's Wiki: https://igoodie.gitbook.io/twitchspawn/twitchspawn-language/tsl-actions
/twitchspawn simulate {event:"donation", amount:1, currency:"EUR"}

**Expected behavior**
 Basically the mod is loaded and read correctly, but Mohist does not run these events which are triggered with events in the live. I ask if you would be able to fix this from the Mohist side, as it works on a very normal forge server.

**Screenshots**

**Info (please complete the following information):**
Mod: twitchspawn_1.12.2-0.4.17

**Comment by NovaPixell** [comment]
Some news?

**Comment by Shawiizz** [comment]
Hi, can you please provide more informations like logs and errors ?

**Comment by HGoo** [comment]
Hello colleagues.
There is a similar problem with https://www.curseforge.com/minecraft/mc-mods/content-creator-integration
Before installing Mohist, everything worked.
Notifications appear but rewards do not spawn (e.g. cows)
Example from config files :

{
  "configs": [
    {
      "types": {
        "message": [
          {
            "name": "Show Twitch Chat",
            "disabled": true,
            "conditions": [],
            "outcomes": [
              {
                "title": "$user says:",
                "subtitle": "$message",
                "titleColor": "$color",
                "type": "toast"
              }
            ]
          },
          {
            "name": "Зомбя",
            "conditions": [
              {
                "variableName": "custom-reward-id",
                "variableResult": "e1794ed8-ecc9-4771-b862-2c175d997852",
                "type": "variableCheck"
              }
            ],
            "outcomes": [
              {
                "sound": "minecraft:entity.illusion_illager.cast_spell",
                "target": "@playerName",
                "volume": 1000.0,
                "pitch": 0.1,
                "type": "sound"
              },
              {
                "toastType": 2,
                "title": "$user Написал",
                "subtitle": "$message",
                "type": "toast"
              },
              {
                "command": "/execute $playerName ~ ~ ~ /summon minecraft:zombie ~ ~1 ~ {CustomName:\"$user\"}",
                "type": "command"
              }
            ],
            "cooldown": 1
          },
          {

**Comment by RogerMinemu** [comment]
Confirmed on mohist-1.12.2-213
Command execution and summon not working.

**Comment by RogerMinemu** [comment]
**Describe the bug**
Mohist don't handle the summon and execute events of TwitchSpawn mod

**To Reproduce**

1) Edit rules.default.tsl
2) Add a rule for example:

`
EXECUTE %/weather thunder%
 ON Twitch Channel Point Reward
 WITH title IS %Zombie%
`

or

`
SUMMON zombie
 ON Twitch Channel Point Reward
 WITH title IS %Zombie%
`

**Expected behavior**
On the first rule it should execute the command /weather thunder (command issued by the player)
On the second one; summon a zombie.


**Screenshots**
![image](https://user-images.githubusercontent.com/8280286/117578535-1e7b4700-b0ef-11eb-9e47-f1bf12d80918.png)
(EXECUTE event shows an error "unknown command"

![image](https://user-images.githubusercontent.com/8280286/117578582-4b2f5e80-b0ef-11eb-9b1f-6310534d9b99.png)
The logs appears like all its good but deff it's not working as intended.


**Info** 
Fresh mohist-1.12.2-213
Mod: twitchspawn_1.12.2-0.8.1

**Comment by RogerMinemu** [comment]
Still happens on mohist-1.12.2-243

**Comment by RogerMinemu** [comment]
Still happens on mohist-1.12.2-249
